### PR TITLE
tasktimer: init at 1.9.4

### PR DESCRIPTION
**Pull request by wahjava**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/tasktimer/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    owner = "caarlos0";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-M88JzcK9HwHeWbQ0McbCF1rIEiKnwh08oEoRck4A/Ho=";
+  };
+
+  vendorSha256 = "sha256-VSrZIM4CnU1AIJs/wDyLxcBd3cGRvpmSD+SYduO7Nlk=";
+
+  postInstall = ''
+    mv $out/bin/tasktimer $out/bin/tt
```
Comment (smancill) [inline pkgs/applications/misc/tasktimer/default.nix:16]: I don't think it's recommended to change the name of the binary.

Comment (smancill) [reply pkgs/applications/misc/tasktimer/default.nix:16]: Ah, I just opened the GitHub repo, it should be `tt`, you are right.

**Comment by smancill** [comment]
Result of `nixpkgs-review pr 157912` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>tasktimer</li>
  </ul>
</details>

**Comment by smancill** [comment]
There is a non-executable `ls_parse.py` file in `$out/bin`:

``` bash
[nix-shell:~/.cache/nixpkgs-review/pr-181597]$  ls -l results/cbmc/bin/
total 50156
-r-xr-xr-x 1 smancill admin 12042112 Dec 31  1969 cbmc
-r-xr-xr-x 1 smancill admin  1539540 Dec 31  1969 crangler
-r-xr-xr-x 1 smancill admin  7648224 Dec 31  1969 goto-analyzer
-r-xr-xr-x 1 smancill admin  5338012 Dec 31  1969 goto-cc
-r-xr-xr-x 1 smancill admin  6368960 Dec 31  1969 goto-diff
lrwxr-xr-x 1 smancill admin        7 Dec 31  1969 goto-gcc -> goto-cc
-r-xr-xr-x 1 smancill admin  3129860 Dec 31  1969 goto-harness
-r-xr-xr-x 1 smancill admin 11280660 Dec 31  1969 goto-instrument
lrwxr-xr-x 1 smancill admin        7 Dec 31  1969 goto-ld -> goto-cc
-r--r--r-- 1 smancill admin    20854 Dec 31  1969 ls_parse.py
-r-xr-xr-x 1 smancill admin  3969628 Dec 31  1969 symtab2gb
```

**Comment by smancill** [comment]
`meta` is missing the `platforms` attribute.

It should also set `mainProgram` to `"tt"`;

**Comment by wahjava** [comment]
Result of `nixpkgs-review pr 157912` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tasktimer</li>
  </ul>
</details>

**Comment by smancill** [comment]
Result of `nixpkgs-review pr 157912` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tasktimer</li>
  </ul>
</details>

**Review by smancill — APPROVED** [review]

**Comment by smancill** [comment]
> `meta` is missing the `platforms` attribute.

Sorry, I messed up, now I checked that if not set it will be set by default by `buildGoModule`, so you can remove it unless there is a reason to specify custom platforms.

**Pull request merged**